design/010-new-dataset-fields/010-new-dataset-fields.md

```diff
@@ -0,0 +1,130 @@
+
+**License**
+
+Data usage license.
+
+* **Reason:** FAIR compliance
+* **Mapping:** [license](https://schema.org/license)
+* **Value:** [URL](https://schema.org/URL) from https://spdx.org/licenses
+* **Example:** `https://schema.org/license: https://spdx.org/licenses/MIT.html`
+
+**Affiliation**
+
+For visibility and citability, affiliations should have a unique identifier. Affiliation already exists as a property of the dataset's creator. Ideally we could add a unique identifier to it. Rather than a property of the Dataset, this may be a property of the creator's `affiliation` and would also be available for projects.
+
+* **Reason:** FAIR compliance + user persona
+* **Mapping:** [creator](https://schema.org/creator) > [affiliation](https://schema.org/affiliation) > [Organization](https://schema.org/Organization) > [identifier](https://schema.org/identifier)
+* **Value:** [URL](https://schema.org/URL) from https://ror.org/
+* **Example:**
+```json
+"http://schema.org/affiliation": [
+    {
+    "@type": "Organization",
+    "name": "EPFL",
+    "identifier": "https://ror.org/02s376052"
+    }
+```
+
+> Note: ROR has a public REST API which could be used to autosuggest records in a text form in the future. More info [here](https://ror.readme.io/docs/create-affiliation-selection-dropdowntypeahead-widgets)
+
+**Topic**
+
+The research field. Ideally, fine grained terms with hierarchical relationships. This would also be relevant for projects.
+
+* **Reason:** User persona + user feedback
+* **Mapping:** [about](https://schema.org/about)
+* **Value:** Maybe [DefinedTerm](https://schema.org/DefinedTerm)
+
+> Note: I am not sure what's the best way of implementing this. [Dataverse](https://demo.dataverse.org/dataverse/demo/search) uses a (short) flat list of terms, whereas [Swiss UBase](https://www.swissubase.ch/en/) uses a fine grained nested list. The closest thing I found is the ["Data Science Education Ontology (DSEO)"](https://fairsharing.org/FAIRsharing.7p0xdg). It has a [Domain](https://bioportal.bioontology.org/ontologies/DSEO/?p=classes&conceptid=http%3A%2F%2Fbigdatau.org%2Fdseo%23domain) class which subclasses owl#Thing.
+
+**Size**
+
+* **Reason:** user feedback
+* **Mapping:** [size](https://schema.org/size)
+* **Value:** [Text](https://schema.org/Text)
+* **Example:** `https://schema.org/size: "130MB"`
+
+> Note: It may not be possible to set it for external datasets.
+
+## Drawbacks
+
+> Why should we *not* do this? Please consider the impact on users,
+on the integration of this change with other existing and planned features etc.
+
+Proliferation of unnecessary fields may impact the performance and size of the knowledge graph, thus negatively affect user-experience.
+
+> There are tradeoffs to choosing any path, please attempt to identify them here.
+
+This may introduce problems with backwards compatibility, since the metadata structure would be modified.
```

Comment:
I think it's more of a problem of how to deal with missing values, which we should consider.